tests/integration/test_candidates.py

```diff
@@ -0,0 +1,123 @@
+import codecs
+import pytest
+import json
+
+import manage
+
+from tests import common
+from webservices import rest, __API_VERSION__
+from webservices.rest import db
+from webservices.resources.candidates import CandidateList
+
+@pytest.mark.usefixtures("migrate_db")
+class IntegrationTestCase(common.BaseTestCase):
```

Comment:
Can we give the test a more descriptive name? I think we already have a `IntegrationTestCase` in `test_integration`